### PR TITLE
Adding stac-check recipe

### PR DESCRIPTION
**Pull request by geomatician**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by geomatician** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your review. Thanks.

File: recipes/stac-check/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  home: https://github.com/stac-utils/stac-check
+  summary: Linting and validation tool for STAC assets
+  license: MIT
+  license_file: LICENSE
+  doc_url: https://github.com/stac-utils/stac-check
+  dev_url: https://github.com/stac-utils/stac-check
+
+extra:
+  recipe-maintainers:
+    - gadomski
```
Comment (beckermr) [inline recipes/stac-check/meta.yaml:53]: This name needs to match your username or we need to ask @gadmoski if they want to maintain.

**Review by beckermr — CHANGES_REQUESTED** [review]
see comment above

File: recipes/stac-check/meta.yaml (hunk 1 above)
Comment (beckermr) [reply recipes/stac-check/meta.yaml]: @jonhealy1 needs to comment themselves here before we can merge

Comment (jonhealy1) [reply recipes/stac-check/meta.yaml]: Hi

**Pull request merged**